### ISSUE TITLE
Need to access via a stored procedure

### ISSUE THREAD

**Issue by ThePrez** [sitemule/ILEusion#10]
Similar to XMLService and Db2Sock, we should have stored procedure access to the JSON-based capabilities. 

This is useful because:
- it allows the application to easily control statefulness by managing Db2 connections
- it defers authentication to the database, inherently supporting Kerberos, etc. 
- it allows remote connections to work without requiring a web server setup on the IBM i (and "always on")
- it allows for other transport mechanisms to be architected (cgi, fcgi, ssh, etc)

**Comment by NielsLiisberg** [comment]
Absolutely!! Actually both ways. Direct calls to the SQL procedure interface will in fact expose existing stored procedures as REST services. However this is a job for the noxDB project rather than ILEusion.

**Comment by WorksOfLiam** [comment]
@NielsLiisberg Is this (program / export function call) something you would like to add to noxDB instead of directly in ILEusion then?
